### PR TITLE
Add studio filter to performer scene list

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: frontend/src/components/checkboxSelect/CheckboxSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        </div>
+      ) : (
+        <div className="d-flex">
+          <Form.Check checked={unselected.includes(option.value)} />
+          <span className="text-muted">{option.label}</span>
+        </div>
+      );
+    return `${
+      unselected.length === 0 ? "All" : unselected.length
+    } studios selected`;
```
Comment (peolic) [inline frontend/src/components/checkboxSelect/CheckboxSelect.tsx]:
```suggestion
    } ${plural} selected`;
```

File: frontend/src/graphql/queries/index.ts [hunk 2]
```diff
@@ -81,6 +86,15 @@ export const usePerformer = (
     skip,
   });
 
+export const useFullPerformer = (
+  variables: PerformerVariables,
+  skip: boolean = false
```
Comment (peolic) [inline frontend/src/graphql/queries/index.ts:91]: This is never executed with `skip=true`, but you might be planning ahead for something.

Comment (ghost) [reply frontend/src/graphql/queries/index.ts:91]: That's just boilerplate, so it's possible to skip the query if necessary.

File: frontend/src/components/checkboxSelect/CheckboxSelect.tsx (hunk 1 above)
Comment (ghost) [reply frontend/src/components/checkboxSelect/CheckboxSelect.tsx]: Thanks, fixed.

**Pull request merged**